themes/bootstrap3/templates/search/facet-list-content.phtml

```diff
@@ -0,0 +1,21 @@
+<ul class="full-facet-list facet__list facet-group" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
+  <?php if ($this->page > 1 && !$this->inLightbox && !$this->ajaxLightbox): ?>
```

Comment:
Why do we need both inLightbox and ajaxLightbox?

Reply:
You're right... i think this PR evolved quite a bit since i started and now ajaxLightbox seems no longer necessary, so i will remove it & only use the default inLightbox here